### PR TITLE
NMS-15040: fix ReST queries for OnmsIpInterface snmpPrimary

### PR DESCRIPTION
**Pull request by amay94**
This was done in Horizon 29.0.0, see NMS-13469.

The issue was reported in Meridian 2021, but it also exists in foundation-2020.

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-13469
* JIRA (Issue Tracker): http://issues.opennms.org/browse/NMS-15040
* Original PR: https://github.com/OpenNMS/opennms/pull/3556

### CODE REVIEW

File: opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/NodeIpInterfacesRestService.java [hunk 1]
```diff
@@ -32,10 +32,7 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import javax.ws.rs.Produces;
+import javax.ws.rs.*;
```
Comment (christianpape) [inline opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/NodeIpInterfacesRestService.java:35]: don't use wildcard imports

File: opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/NodeIpInterfacesRestService.java [hunk 2]
```diff
@@ -48,10 +45,7 @@
 import org.opennms.core.criteria.Alias.JoinType;
 import org.opennms.core.criteria.CriteriaBuilder;
 import org.opennms.netmgt.dao.api.IpInterfaceDao;
-import org.opennms.netmgt.model.OnmsIpInterface;
-import org.opennms.netmgt.model.OnmsIpInterfaceList;
-import org.opennms.netmgt.model.OnmsMetaDataList;
-import org.opennms.netmgt.model.OnmsNode;
+import org.opennms.netmgt.model.*;
```
Comment (christianpape) [inline opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/NodeIpInterfacesRestService.java:48]: don't use wildcard imports

File: opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/NodeRestService.java [hunk 3]
```diff
@@ -34,10 +34,7 @@
 import java.util.Set;
 import java.util.stream.Collectors;
 
-import javax.ws.rs.GET;
-import javax.ws.rs.Path;
-import javax.ws.rs.PathParam;
-import javax.ws.rs.Produces;
+import javax.ws.rs.*;
```
Comment (christianpape) [inline opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/NodeRestService.java:37]: don't use wildcard imports

File: opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/NodeRestService.java [hunk 4]
```diff
@@ -54,6 +51,7 @@
 import org.opennms.netmgt.dao.api.MonitoringLocationDao;
 import org.opennms.netmgt.dao.api.NodeDao;
 import org.opennms.netmgt.events.api.EventProxy;
+import org.opennms.netmgt.model.OnmsMetaData;
```
Comment (christianpape) [inline opennms-webapp-rest/src/main/java/org/opennms/web/rest/v2/NodeRestService.java:54]: don't use wildcard imports

**Review by christianpape — CHANGES_REQUESTED** [review]
Only small changes needed.

**Review by aramosvizcarra — APPROVED** [review]
LGTM

**Pull request merged**